tools/IndexRebuilder/app.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <connectionStrings>
+    <add name="Database" connectionString="server=.;database=tempdb;integrated security=true"/>
+  </connectionStrings>
+  <appSettings>
+    <add key="IsSharded" value="false"/>
+    <add key="Threads" value="16"/>
+    <add key="IndexToKeep" value="IX_Resource_ResourceTypeId_ResourceId_Version"/>
+    <add key="EventLogQuery" value="SELECT TOP 100 * FROM EventLog (NOLOCK) WHERE EventDate > dateadd(hour,-24,getUTCdate()) ORDER BY EventDate DESC, EventId DESC"/>
```

Comment:
Just curious, why does `EventLogQuery` need to be configurable?

Reply:
This is just an example query to facilitate debugging. Our code analyzers were complaining that string should not be part of code, but in config.